configure.ac

```diff
@@ -361,6 +361,22 @@ case $host in
    *linux*)
      TARGET_OS=linux
      ;;
+   *dragonfly*)
```

Comment:
I use Dragonfly and Mac.  The others are obvious too, if you look at LevelDB makefile, and it has been confirmed to fix FreeBSD build of BU by a third party; check  our issues list.